mifosng-android/build.gradle

```diff
@@ -11,7 +11,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.2'
+        classpath 'com.android.tools.build:gradle:3.1.3'
```

Comment:
Changes like these need separate PR (for the separate issue). Please don't include in this fix.

Reply:
Sorry this was done automatically by android studio